src/installment-sale/lib/logic.ergo

```diff
@@ -17,70 +17,81 @@ namespace org.accordproject.installmentsale
 import org.accordproject.cicero.runtime.*
 import org.accordproject.money.MonetaryAmount
 
+define function round(x:Double) {
```

Comment:
Hmm, how should that be done? 
For the last installment, it is easy to create an `enforce` statement because we know exactly how much it should be done. 
For the other payments, should it be done by using the `roundn` function for all of the values, like before_interest, balance, and so on?

Reply:
I was thinking it could be done using the same `roundn` function used here, but it's not mandatory. Just wondered why do it in one place, but not the rest of the code?